### DISCUSSION TITLE
Q1: App not running

### DISCUSSION THREAD

**Discussion by Anujsewani** [swetank01/open-microapp#19] [Q&A] [answered]
Hi @swetank01 
After following the steps from readme.md file, the app is not running gives 404 not found error 
192.168.29.69 - - [16/Nov/2023 16:09:23] "GET / HTTP/1.1" 404 -
192.168.29.69 - - [16/Nov/2023 16:12:23] "GET /users HTTP/1.1" 404 -
![Screenshot from 2023-11-10 17-38-28](https://github.com/swetank01/open-microapp/assets/51258892/fc13f49f-b2ba-4206-863d-3b086317486b)

**Comment by swetank01** [comment, marked as answer]
@Anujsewani   basically this is an users api service which seems working fine!!! 
take a look into this [link](https://github.com/swetank01/open-microapp/blob/dev/Docs/DevDocs/testing/user-authenciation-api.md)  

Also if you look into the code [here](https://github.com/swetank01/open-microapp/blob/dev/user_service.py)
there is no '/users' endpoint.. so just try to read user_service.py code you will get your answer
